doc/source/rllib/new-api-stack-migration-guide.rst

```diff
@@ -22,29 +22,21 @@ RLlib classes and code to RLlib's new API stack.
     Note, though, that the Ray team continues to  design RLlib to be framework-agnostic.
 
 
-Change your AlgorithmConfig
----------------------------
+Check your AlgorithmConfig
+--------------------------
 
-RLlib turns on the new API stack by default for all RLlib algorithms. To deactivate it, use the `api_stack()` method
-in your `AlgorithmConfig` object like so:
+RLlib turns on the new API stack by default for all RLlib algorithms.
 
-.. testcode::
+.. note::
+    To **deactivate** the new API stack and switch back to the old one, use the
```

Comment:
We should leave here a comment for when we see the old stack being deprectaed and not supported anymore. I see way too many users on the discussion forums to go with the old stack, even when starting their journey. I always mention that the old stack will be deprecated soon, but without a date.

Reply:
Yup! Let's change the "new stack" header everywhere describing that they should ONLY use the new stack from here on (unless they have very good reasons to still use the old one). ...